api/src/main/java/org/openmrs/module/fhir2/api/dao/FhirUserDao.java

```diff
@@ -13,7 +13,7 @@
 
 import org.openmrs.User;
 
-public interface FhirUserDao {
+public interface FhirUserDao extends FhirDao<User> {
 	
 	User getUserByUuid(@NotNull String uuid);
```

Comment:
We can actually probably get rid of this method since, to be consistent, it should be called `get()`.